### ISSUE TITLE
Allow Product Update emails to attach current version

### ISSUE THREAD

**Issue by carter2422** [easydigitaldownloads/fes-product-updates#4]
Since product emails will mostly, I assume, be used for notifying customers of new product versions I think we should give vendors the option to `Include download link for latest version`. This would allow the customer to get their update easily, instead of being left wondering how to get the update. In particular, this will help vendors that don't have the best grammar or English. 

So as to also bring the customer back to the market, though, it may be good to just auto-link them to their customer dashboard.

**Comment by pippinsplugins** [comment]
Yep! Product Updates already supports email tags, just like standard purchase receipts, so we just need to provide vendors with a list of the tags they can use.

**Comment by carter2422** [comment]
Excellent!
